Committing changes as reviewed by Alberto through pull request comments.

Seshan01 opened the pull request:
Name of the Tank and Junction entities have been changed to WasteWaterTank and WasteWaterJunction.
Removal of sub-properties in the base schema.
Fixing of type error in OffGasStack entity.

Seshan01 committed:
Committing changes as reviewed by Alberto through pull request comments.
Name of the Tank and Junction entities have been changed to WasteWaterTank and WasteWaterJunction.
Removal of sub-properties in the base schema.
Fixing of type error in OffGasStack entity.

albertoabellagarcia commented on wastewater/dataModel.WasteWaterTreatment/WasteWaterTreatment-schema.json:
```diff
@@ -1,80 +1,4 @@
 {
```
missing the header of this file

Seshan01 replied:
Hi Alberto,

Thanks I have added the header of the file. I have a question regarding the base schema. What properties are we needed to define in the base schema? From all the examples, it is not clear to me. The only one that I can understand is the Commons definitions.

Until now I was under the impression, we need to define all possible properties of the various entities within the wastewater data models (Tank, Blower, OffGasStack etc.) in the base schema as well. But from the examples, that does not seem to be the case?

albertoabellagarcia replied:
The properties have to be defined only in one place (either at common-schema.json, in a shared schema like [this](https://github.com/smart-data-models/dataModel.WaterNetworkManagement/blob/master/WaterNetworkManagement-schema.json) and then call through a $ref or directly in the data model).
In the case that they are defined in two places this would be an error. If you find any please let me know.

albertoabellagarcia reviewed (changes requested):
fix the wastewaterTreatment-schema.json (see this example [https://github.com/smart-data-models/dataModel.Weather/blob/master/weather-schema.json](https://github.com/smart-data-models/dataModel.Weather/blob/master/weather-schema.json)

Seshan01 committed:
Committing minor changes in the Blower and OffGasStack schemas.
Committing a new schema for the WasteWaterDevice. Most properties are common to Device model. New properties to be added to support the wastewater use case.

Seshan01 committed:
Committing minor change in the WasteWaterJunction schema.

albertoabellagarcia commented on wastewater/dataModel.WasteWaterTreatment/Blower/schema.json:
```diff
@@ -1,9 +1,9 @@
     "title": "WasteWaterTreatment - Blowers",
-    "description": "This entity contains an harmonised description of a Blower made for the Wastewater treatment domain. The entity represents a Blower that are used for aerationpurposes in the wastewater treatment process. Important parameters are measured to regulate andmeasure the amount of airflow is being provided to the aeration tank in the bioreactor. Energy consumption of a blower is also ...
+    "description": "This entity contains an harmonised description of a Blower made for the Wastewater treatment domain. The entity represents a Blower that are used for aeration purposes in the wastewater treatment process. Important parameters are measured to regulate and measure the amount of airflow is being provided to the aeration tank in the bioreactor. Energy consumption of a blower is al ...
     "type": "object",
```
"type": "object" has to be removed

albertoabellagarcia reviewed (changes requested):
minor changes, please correct.
